internal/provider/resource_vault_cluster_perf_replication_test.go

```diff
@@ -0,0 +1,330 @@
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const (
+	hvn1ResourceName           = "hcp_hvn.hvn1"
+	hvn2ResourceName           = "hcp_hvn.hvn2"
+	primaryVaultResourceName   = "hcp_vault_cluster.c1"
+	secondaryVaultResourceName = "hcp_vault_cluster.c2"
+)
+
+func setTestAccPerformanceReplicationE2E(tfCode string) string {
+	return fmt.Sprintf(`
+resource "hcp_hvn" "hvn1" {
+	hvn_id            = "test-perf-hvn-1"
```

Comment:
These IDs also need to be timestamped, with max 36 characters.

Reply:
Done